[2023-05-04T04:31:44.497+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: evidently_test_suite.load_data scheduled__2023-05-03T00:00:00+00:00 [queued]>
[2023-05-04T04:31:44.513+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: evidently_test_suite.load_data scheduled__2023-05-03T00:00:00+00:00 [queued]>
[2023-05-04T04:31:44.514+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T04:31:44.514+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2023-05-04T04:31:44.514+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T04:31:44.542+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_data> on 2023-05-03 00:00:00+00:00
[2023-05-04T04:31:44.571+0000] {standard_task_runner.py:55} INFO - Started process 146 to run task
[2023-05-04T04:31:44.578+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'evidently_test_suite', 'load_data', 'scheduled__2023-05-03T00:00:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/evidently_test_suite.py', '--cfg-path', '/tmp/tmp13v981md']
[2023-05-04T04:31:44.581+0000] {standard_task_runner.py:83} INFO - Job 93: Subtask load_data
[2023-05-04T04:31:44.710+0000] {task_command.py:389} INFO - Running <TaskInstance: evidently_test_suite.load_data scheduled__2023-05-03T00:00:00+00:00 [running]> on host 6c2147fba45d
[2023-05-04T04:31:44.887+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=evidently_test_suite
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2023-05-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-03T00:00:00+00:00
[2023-05-04T04:31:59.460+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-04T04:31:59.481+0000] {taskinstance.py:1322} INFO - Marking task as SUCCESS. dag_id=evidently_test_suite, task_id=load_data, execution_date=20230503T000000, start_date=20230504T043144, end_date=20230504T043159
[2023-05-04T04:31:59.588+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-05-04T04:31:59.643+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
